### ISSUE TITLE
set up jest for testing and istanbul for coverage check

### ISSUE THREAD

**Issue by rae-ralston** [rachel-ftw/lyft-to-the-park#19]
- [ ] Install and set up jest
- [ ] write a single test that passes
- [ ] install and set up istanbul
- [ ] get a code coverage report
- [ ] update the package.json with appropriate scripts
- [ ] update the readme with instructions for future contributors on how to run the tests.

**Comment by Atul9** [comment]
Hi @rachel-ftw, I would like to contribute to this project. I tried to set it up. I provided my number for lyft credentials but I didn't get any message from them. I think they send messages only to US mobile numbers.

**Comment by rae-ralston** [comment]
Thanks for jumping on the project! 😍

As it stands, the Lyft API portion of the application is still under development, so you should still be able to contribute without it. Did you get the google geocoding & NPS API keys without trouble? They're the important ones for now. 

To complete this issue I'd install and set up jest & istanbul and submit an PR with a single test passing & a test coverage report. issues #21 & #20 cover getting into test writing more.  I just filled out this issue with more detail (see above).

To continue to contribute once the lyft portion is incremented you could get a VOIP US number to confirm with that. You could probably get something like https://voice.google.com/ 
Let me know if I can help you with that! 

Thanks!

**Comment by Atul9** [comment]
Yes, I got google geocoding & NPS API keys. I have opened up a PR https://github.com/rachel-ftw/lyft-to-the-park/pull/25
To setup jest in continuous integration(CI) we need to set up this project in Travis or some other CI website. Can you set up this project in a CI/CD(continuous delivery) service?

**Comment by rae-ralston** [comment]
Thanks for logging the issue.
Yep. I'll be away from a computer for the next 24-ish hours, but I'll set it up on Sunday.
Thanks @Atul9! 🎉
